### ISSUE TITLE
Option to `preventDefault` on events

### ISSUE THREAD

**Issue by nathanboktae** [nathanboktae/aria-listbox#1]
An option to `preventDefault` on the keyboard and mouse events that `aria-listbox` should be added to prevent scrolling and such. However to plugin well with other components, they may also want to capture them so it needs to not capture events in that case, as it currently is.

Should this be the default, though?
